app/code/Magento/MediaGalleryUi/composer.json

```diff
@@ -12,7 +12,8 @@
         "magento/module-media-gallery-metadata-api": "*",
         "magento/module-media-gallery-synchronization-api": "*",
         "magento/module-media-content-api": "*",
-        "magento/module-cms": "*"
+        "magento/module-cms": "*",
+        "magento/module-media-gallery": "*"
```

Comment:
Probably it's better to move the ACL to API module to have the dependency on API module only